vsts/pipelines/templates/_buildParallel.yml

```diff
@@ -1,15 +1,12 @@
 parameters:
```

Comment:
I wonder if we should call this file _buildTemplate.tml now, since parallel is an implementation detail, and build template is what it really is?

Reply:
I agree … I will be sending another PR soon to address this